src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Executor
+{
+    public class Executor
+    {
+        public Setup openFile(string filePath)
+        {
+            string jsonFile = string.Empty;
+            if (File.Exists(filePath))
+            {
+                jsonFile = File.ReadAllText(filePath);
+                Setup jsonSetup = JsonConvert.DeserializeObject<Setup>(jsonFile);
+                return jsonSetup;
+            }
+            return null;
+        }
+
+        public void usage()
+        {
+            Console.WriteLine("Executor tool");
+            Console.WriteLine("Usage: executor [config file] [command]");
+            Console.WriteLine("  Config file:      json file that contains the properties and the commands to be run. (Required)");
+            Console.WriteLine("Command: [executor command]|[config command]");
+            Console.WriteLine("  Executor command:   Executor tool commands.");
+            Console.WriteLine("         info       Provides information about the executor tool.");
+            Console.WriteLine("         help       Provides information about the content of the config file. ");
+            Console.WriteLine("  Config command:   Config file specific commands.");
+        }
+                
+        public static void Main(string[] args)
+        {
+            int argsLength = args.Length;
+            if (argsLength > 0)
+            {
+                Executor executor = new Executor();
+                Setup jsonSetup = executor.openFile(args[0]);
+                if (jsonSetup != null)
+                {
+                    //will be passed by executor.cmd/sh? we only need to now if it is in windows or non-windows.
+                    string os = "windows";
+                    //default right now.
+                    string command = "simple";
+                    bool end = false;
+                    for (int i = 1; i < argsLength && !end; i++)
+                    {
+                        switch (args[i])
+                        {
+                            case "info":
+                                executor.usage();
+                                end = true;
+                                break;
+                            case "help":
+                                Console.WriteLine("Usage: build [command]:");
```

Comment:
I agree with the two of you that build is not the best word for all the cases we want to cover.
What I see from the pizza example is that we will maintain a build.cmd script that will build everything (by calling the executor) but that doesn't mean that build is the only entry point for the executor.
In that case we need to agree on a word :)